### PR TITLE
Use containerized builds on Travis

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,3 +21,5 @@ script:
 
 notifications:
   irc: "chat.freenode.net#openshift-dev"
+
+sudo: false
```
Comment (liggitt) [inline .travis.yml]: I thought this was the default. Is it not?

**Comment by smarterclayton** [comment]
Broken because we need ifconfig which the image doesn't have.  We need ifconfig for hack/test-cmd to work on a mac.  I hate computers.

File: .travis.yml (hunk 1 above)
Comment (smarterclayton) [inline .travis.yml]: No, it defaults to true because they used to run in vms.

**Comment by smarterclayton** [comment]
[merge]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**